### ISSUE TITLE
iOS Support

### ISSUE THREAD

**Issue by Auto72** [Dtronix/PDFiumCore#17]
Is It possible to include the iOS support?

https://www.nuget.org/packages/bblanchon.PDFium.iOS/

**Comment by DJGosnell** [comment]
I have not done any iOS development before.  I know that .NET can run on it, I just don't have experience in the area.  I'll look into this, but it is a low priority right now unless you can help with a PR.

**Comment by Auto72** [comment]
Actually, I tried to add the following Nuget packages to the PDFiumCore project:

"bblanchon.PDFium.Android"
"bblanchon.PDFium.iOS"

and run in sequence:

1) download_package.sh
2) CreateBindingsPackage.bat
3) build_test_publish.ps1

I've got the generated PDFiumCore.121.0.6164.nupkg file from the "artifacts" folder and used in my .NET MAUI test project as OFF Line package. On Android it works, but on iOS it doesn't work.

On iOS I see the .NET splash screen for few moments and then the App crashes :-(
The iOS Program.Main() is not event reached.
